.github/workflows/conformance-gateway-api.yaml

```diff
@@ -31,8 +66,17 @@ env:
   gateway_api_version: v0.7.0
   metallb_version: 0.12.1
   timeout: 5m
+  check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
I don't think `check_url` is used anymore, is it? We should probably just remove it from other workflows if it's still there.